pyproject.toml

```diff
@@ -176,5 +176,8 @@ directory = "coverage/html"
 output = "coverage/index.xml"
 #=== End: PyTest and Coverage
 
-[flake8]
-max-line-length = 120
+[tool.flake8]
+max-line-length = 100
```

Comment:
```suggestion
max-line-length = 120
```

Reply:
Note: flake8 doesn't support `pyproject.toml` configuration; this requires an additional [plugin](https://github.com/john-hen/Flake8-pyproject). We don't actively use `flake8` anyway, so perhaps we should just delete this.

Reply:
I was actually hoping to introduce flake8 later as well 😬😂.
The plugin is just a simple extra dependency on the `.[test]` install and then it works. I already use it locally.